### PR TITLE
Docs: fix highlight border radius

### PR DESCRIPTION
**Pull request by julien-deramond**
### Description

This PR tries to fix the highlight border radius in our docs coming from https://github.com/twbs/bootstrap/commit/20ab8219a0caca784483e93083cde269055d0f3a.

⚠️ But maybe there is a smarter/better way to do it. IMO the rendering in this PR is better but the values are not exactly the same depending on the cases; sometimes 7px, sometimes 8px.

#### Issues

(_Shoutout to @Hideoo for catching it_)

Upper border radius of the "simple" highlight is straight in our _main_ branch contrary to the live site.

Screenshot 2023-02-25 at 08 31 00

Screenshot 2023-02-25 at 08 32 08

But in some cases with more complex rendering, it was OK like in https://twbs-bootstrap.netlify.app/docs/5.3/components/accordion/#example

OK but still with a small issue (in https://twbs-bootstrap.netlify.app/docs/5.3/components/accordion/#flush for example):

Screenshot 2023-02-25 at 08 33 16

### Type of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] I have updated the documentation accordingly
- (N/A) I have added tests to cover my changes
- [x] All new and existing tests passed

#### Live previews

- https://deploy-preview-38114--twbs-bootstrap.netlify.app/

Plus examples or more complex use cases in:

- https://deploy-preview-38114--twbs-bootstrap.netlify.app/docs/5.3/getting-started/introduction/
- https://deploy-preview-38114--twbs-bootstrap.netlify.app/docs/5.3/components/accordion/#example
- https://deploy-preview-38114--twbs-bootstrap.netlify.app/docs/5.3/components/accordion/#flush

### CODE REVIEW

File: site/assets/scss/_component-examples.scss [hunk 1]
```diff
@@ -401,7 +400,7 @@
 
 .bd-scss-docs {
   .highlight-toolbar {
-    @include border-top-radius(calc(var(--bs-border-radius) + 1px));
+    @include border-top-radius(calc(var(--bs-border-radius) - 1px));
```
Comment (louismaximepiton) [inline site/assets/scss/_component-examples.scss:403]:
```suggestion
    @include border-top-radius(var(--bs-border-radius));
```

Looks better to me with the `border-radius` increased in `.bd-code-snippet` 🤷

**Comment by julien-deramond** [comment]
Thanks @louismaximepiton I'll try what's suggested.
Btw, there's another issue in the same spirit with the bottom border-radius as well in https://twbs-bootstrap.netlify.app/docs/5.3/content/reboot/#blockquote:
Screenshot 2023-03-02 at 20 21 10

I'll try to check if this PR can fix it as well. Otherwise, I'll create another issue.

**Comment by mdo** [comment]
@julien-deramond I have another pass at something in #38491, along with a new docs page to help us test these things better in the future.

**Comment by julien-deramond** [comment]
Superseded by #38491
Closing it

**Pull request closed without merging**